src/engine/Shopware/Plugins/Community/Frontend/Paymentwall/Controllers/Frontend/Brick.php

```diff
@@ -0,0 +1,111 @@
+<?php
+if (!class_exists('Paymentwall_Config'))
+    require_once(Shopware()->OldPath() . "engine/Library/paymentwall/lib/paymentwall.php");
+
+define('BASE_URL', Shopware()->Shop()->getBaseUrl());
```

Comment:
It may conflict with other Shopware's module
Pls move it into class or use their function